src/loaders/schema/schema-from-export.ts

```diff
@@ -8,7 +8,13 @@ export class SchemaFromExport implements SchemaLoader {
   canHandle(pointerToSchema: string): boolean {
     const fullPath = isAbsolute(pointerToSchema) ? pointerToSchema : resolvePath(process.cwd(), pointerToSchema);
 
-    return isValidPath(pointerToSchema) && existsSync(fullPath) && extname(pointerToSchema) !== '.json';
+    if (isValidPath(pointerToSchema) && existsSync(fullPath) && extname(pointerToSchema) !== '.json') {
+      const exports = require(fullPath);
```

Comment:
I think it might break stuff when it's a `ts` file and no `ts-node/register` is used

Reply:
An example:

```ts
export const typeDefs: DocumentNode = gql`
  type Query { ... }
`;
```

Reply:
Can you please elaborate? I used the very same logic used in the handler.